nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -100,6 +100,8 @@
 
 - The binary of the package `cloud-sql-proxy` has changed from `cloud_sql_proxy` to `cloud-sql-proxy`.
 
+- Emacs was upgraded from version 28 to 29. This includes many major aditiions, most notably the pgtk variant for Wayland users which is available under the attribute `emacs-pgtk`.
```

Comment:
```suggestion
- Emacs was upgraded from version 28 to 29. This includes many major additions, most notably the pgtk variant for Wayland users which is available under the attribute `emacs-pgtk`.
```

Reply:
Correct me if I'm wrong, but pgtk is not restricted to Wayland users (although it is very useful for them).